### DISCUSSION TITLE
Download and Install Scheme Interface

### DISCUSSION THREAD

**Discussion by PatrickSJacobs** [NanoComp/meep#2618] [Q&A]
Hello!

I am trying to install an older version of MEEP with the Scheme interface but I cannot find any README file in Read the Docs or here in the GitHub. Does anyone know where I can install the older version of MEEP. 

Any help would be appreciated!

**Comment by stevengj** [comment]
You just have to compile it from source, making sure that you have Guile (Scheme) installed first.  See [these instructions](https://meep.readthedocs.io/en/latest/Build_From_Source/).
